infra/src/test/scala/ch/epfl/scala/services/storage/sql/SqlRepoTests.scala

```diff
@@ -1,91 +1,86 @@
 package ch.epfl.scala.services.storage.sql
 
-import scala.concurrent.Await
-import scala.concurrent.Future
-import scala.concurrent.duration.Duration
-import scala.util.Success
-import scala.util.Try
+import scala.concurrent.ExecutionContext
 
-import ch.epfl.scala.services.storage.sql.Values
-import doobie.scalatest.IOChecker
-import org.scalatest.BeforeAndAfterAll
+import ch.epfl.scala.index.Values
 import org.scalatest.funspec.AsyncFunSpec
 import org.scalatest.matchers.should.Matchers
+import scala.concurrent.ExecutionContextExecutor
 
-class SqlRepoTests
-    extends AsyncFunSpec
-    with Matchers
-    with IOChecker
-    with BeforeAndAfterAll {
-  private val db = Values.db
-  val transactor = Values.xa
+class SqlRepoTests extends AsyncFunSpec with BaseDatabaseSuite with Matchers {
 
-  override def beforeAll(): Unit = db.migrate().unsafeRunSync()
+  // TODO: use a fixed pool execution context
```

Comment:
for tests I don't think it's an issue!

Reply:
As long as one has more than one core it should work. But it is always good to know exactly how much cores are needed.